docs/parse_argparse.py

```diff
@@ -117,38 +162,9 @@ def path_dir_docs_scripts(fpath):
     return (rst_dirpath)
```

Comment:
In my opinion, it would be better to keep it as it is because to run `write usage` i would need `write_usage(basename, rst_fpath, wrapped_lines)` . 
Also, right now it is more like one function is doing one job, returning `rst_fpath` from a function whose job is to make `directories` would be a bit confusing. Right now it looks like a better division of tasks
What do you think?